### DISCUSSION TITLE
Your Team encountered an unknown problem. Please reach out to our support team for details. Learn More

### DISCUSSION THREAD

**Discussion by momo-mnsjtxy** [vercel/community#6063] [Account Issue] [answered]
### Summary

Your Team encountered an unknown problem. Please reach out to our support team for details. [Learn More](https://vercel.com/help)

### Steps to Reproduce

Your Team encountered an unknown problem. Please reach out to our support team for details. [Learn More](https://vercel.com/help)

**Comment by amyegan** [comment]
Hey @momo-mnsjtxy. Please share more about where you saw this error and what you were trying to do before the error was thrown. 🙏

**Reply by oceanfrost** [reply]
This prompt also appears when I create a project and clone the template. @
![Screenshot_2024-03-09-13-05-25-790_com microsoft emmx](https://github.com/vercel/community/assets/95454779/4bd66683-0f89-4730-ac52-b37977b00f55)

**Reply by mctn6** [reply]
@amyegan , same issue with [6027](https://github.com/orgs/vercel/discussions/6027)

**Reply by swarnava** [reply]
@momo-mnsjtxy @Ocean456 @mctn6 Can you try deploying now and let us know if you are still seeing this issue?

**Reply by mctn6** [reply]
solved in my case, thank you so much @swarnava , @amyegan

**Reply by oceanfrost** [reply]
Thanks, my problem is solved.@swarnava

**Comment by momo-mnsjtxy** [comment]
![Screenshot_20240310_093348_com huawei browser](https://github.com/vercel/community/assets/122842784/2bcf70f9-aa28-4f52-800c-69ebede8455d)

**Reply by momo-mnsjtxy** [reply]
@amyegan

**Reply by nadeeraka** [reply]
same hear

**Reply by swarnava** [reply]
@nadeeraka can you try deploying again?

**Comment by momo-mnsjtxy** [comment]
hi @amyegan

**Comment by amyegan** [comment]
Thanks for reporting this. It was also reported in https://github.com/vercel/community/discussions/6027. Let's move over there to keep the conversation in one place.

**Comment by swarnava** [comment]
This issue should be fixed for all the users reported in this thread . Can you try again and let us know how it goes?

**Reply by nadeeraka** [reply]
@swarnava it resolve thanks for the support and response

**Comment by unoejk** [comment]
@amyegan
Hello. Same problem. Can u fix me too?
my acc on vercel:
login/ unoejk
mail/ unoejk@gmail.com

**Reply by swarnava** [reply]
This should be fixed now

**Reply by unoejk** [reply]
It works, thanks)

**Comment by alikaz3mi** [comment]
@amyegan
Hello. Same problem. Can u fix me too?


email: alikaz3mi1374@gmail.com

**Reply by alikaz3mi** [reply]
@amyegan
Could you check my issue?

**Comment by KorryKatti** [comment]
same problem here

**Comment by Shaju06** [comment]
I'm also receiving same error while deploying my nextjs app to vercel

**Comment by shravanshetty1** [comment]
same problem...

**Comment by amyegan** [comment, marked as answer]
The problem has been resolved. Everyone here should be able to deploy now.

Thank you all for your patience and understanding while we worked to resolve this tricky issue 🙇

**Reply by reenamyka** [reply]
I'm still receiving the same error when I deploy next.js to vercel

**Reply by amyegan** [reply]
Hey @reenamyka. I responded in https://github.com/vercel/community/discussions/6027#discussioncomment-9439020

**Reply by shravanshetty1** [reply]
@amyegan just checked it today, and it has not been resolved for me

**Reply by amyegan** [reply]
@shravanshetty1 Thanks for letting me know. Can you give it another try? It should work now

**Reply by shravanshetty1** [reply]
@amyegan works for me now

**Reply by amyegan** [reply]
If you still see this error, please go to https://github.com/vercel/community/discussions/6027 for the latest updates

**Comment by reenamyka** [comment]
Same problem
